### DISCUSSION TITLE
Uploaded beatmap didnt show cover and card background properly

### DISCUSSION THREAD

**Discussion by Furozzu** [ppy/osu#17944] [Q&A] [answered]
as said on title, beatmap didnt show card.jpg and cover.jpg properly. when i press f12, it show code 403 access denied and when i want to access said cover.jpg, it appear like this 
![image](https://user-images.githubusercontent.com/29659076/164879107-396bef98-1ed0-42b1-828e-afdc0e1d7682.png) (with RequestID and HostID below that line)

the card:
![image](https://user-images.githubusercontent.com/29659076/164879317-3ac69806-2f1b-4fb0-bae0-1a8c2acf550c.png)
the site:
![image](https://user-images.githubusercontent.com/29659076/164879370-4f219b38-3368-4319-b269-6005837b54ee.png)
f12 console screen:
![image](https://user-images.githubusercontent.com/29659076/164879406-11fb6543-b736-49d8-80b6-d8f077a34abb.png)

this problem not only affect my map, but it affect others that did upload beatmap around that time (i did check this map https://osu.ppy.sh/beatmapsets/1748659 and try to subtract and add beatmap id by 1-100 and problem still happen

am i need to do something in my client side or is it problem from server side? please help

**Comment by peppy** [comment, marked as answer]
Please wait 12 hours or so. It will fix itself.
